src/main/java/org/dita/dost/module/MoveLinksModule.java

```diff
@@ -60,6 +62,8 @@ public AbstractPipelineOutput execute(final AbstractPipelineInput input) throws
 
             final XsltTransformer transformer = xsltCompiler.compile(new StreamSource(styleFile)).load();
             transformer.setErrorListener(toErrorListener(logger));
+            transformer.setURIResolver(new DelegatingURIResolver(CatalogUtils.getCatalogResolver(), job.getStore()));
```

Comment:
The `DelegatingURIResolver` allow chaining XML catalog resolver and `Store` (which implements `URIResolver`), so that in the end `Store` will return the `Source` that is used to read the resource.